lib/engines/content_object_store/app/components/content_object_store/content_block/document/show/document_timeline_component.rb

```diff
@@ -27,9 +27,8 @@ def first_created_edition
   end
 
   def time_html(date_time)
-    formatted_timestamp = date_time.strftime("%d %B %Y at %I:%M%P")
```

Comment:
I wonder if we ought to go the other way here and be consistent by using `long_ordinal`?

I tried to look [for guidance on time](https://www.gov.uk/guidance/style-guide/a-to-z#times) and [dates](https://www.gov.uk/guidance/style-guide/a-to-z#dates) (just in case) but there wasn't anything helpful. Did you find anything?

In addition, adding in `human_readable` as a new time format alongside `long_ordinal` does make it harder to understand which format to use to be consistent. Unless of course we are highly confident that this is in keeping with the style guide?

Reply:
I think you're right. Keeps it simple and consistent then